Strategy11/formidable-forms
# Change conditional logic row's value select element type conditionally

Related issue: https://github.com/Strategy11/formidable-pro/issues/5307
Related PR: https://github.com/Strategy11/formidable-pro/pull/5338

### Test procedure
1. Create a form and add an option field like dropdown field and another field, like a text field.
2. Start adding a conditional logic to the text field and select the dropdown field as value source and select comparison among `contains`, `does not contain`, `starts with` or `ends with`
3. Confirm that the control in which you define the match value is changed to text input.
4. Enter a value in the text field and save your form.
5. Confirm that the value select control is still a text input even after refreshing your page.
6. Preview your form and then test the conditional logic works as expected.
7. Get back to the form builder and change the comparison type to something else except those mentioned in step 2 above.
8. Save and preview your form and confirm the conditional logic works as expected.
